lib/util/git/index.ts

```diff
@@ -221,6 +222,7 @@ async function fetchBranchCommits(preferUpstream = true): Promise<void> {
       .forEach(([sha, ref]) => {
         config.branchCommits[ref.replace('refs/heads/', '')] =
           sha as LongCommitSha;
+        // TODO: branchUpdateDate cannot be set withou fetching the commit, check if this is needed
```

Comment:
For Gerrit this PR is working great. But I have not tested for other platforms.

The risk is whether `branchUpdateDate` is not yet set when the count is performed.

If I were able to set the update date in this function, there would be no risk, but it's impossible to get the commit date without actually fetching the commit.

Thoughts?